hack/testing/rsyslog/roles/rsyslog/tasks/main.yaml

```diff
@@ -0,0 +1,159 @@
+---
+- name: Install required packages
+  yum:
+    name: '{{ item }}'
+    state: 'present'
```

Comment:
Should we also try to update the packages or only have them present?

Reply:
update too

Reply:
So this should be updated to latest

Reply:
So, this task is good, right?
Had I better change the "name" to "Install/Update required packages"?

Reply:
I see.  Done.  Thanks!